### DISCUSSION TITLE
官方的社交登录配置变量为什么没有了？

### DISCUSSION THREAD

**Discussion by AkarinLiu** [walinejs/waline#911] [Q&A] [answered]
想配置Github社交登录，但是在官方文档中找不到官方的社交登录配置变量，不知道是否被移除了。@lizheming

**Comment by lizheming** [comment, marked as answer]
不需要配置了，Waline 将社交登录部分收敛成了一个 Serverless 服务，官方默认提供，这样就不需要每个用户都配置了。如果需要自定义的话可以设置 `OAUTH_URL` 配置 https://waline.js.org/reference/server.html#%E4%B8%BB%E8%A6%81%E9%85%8D%E7%BD%AE

**Comment by AkarinLiu** [comment]
你们应该把它写到文档里面

**Reply by lizheming** [reply]
给你贴的就是文档地址
